cwltool/sandboxjs.py

```diff
@@ -101,9 +100,9 @@ def term():
     tm = threading.Timer(timeout, term)
     tm.start()
 
-    stdin_buf = StringIO(json.dumps(fn) + "\n")
-    stdout_buf = StringIO()
-    stderr_buf = StringIO()
+    stdin_buf = BytesIO(json.dumps(fn) + "\n")
+    stdout_buf = BytesIO()
+    stderr_buf = BytesIO()
 
     completed = []  # type: List[Union[cStringIO.InputType, cStringIO.OutputType]]
```

Comment:
I updated `schema_salad` and `cwltool` for `mypy` 0.470 (the project was renamed from `mypy-lang`). With that in place `type: List[BytesIO]` works for me. If you'll merge in from master then I think we'll be done with this PR.